akka-sse/src/test/scala/de/heikoseeberger/akkasse/ServerSentEventParserSpec.scala

```diff
@@ -27,20 +27,21 @@ class ServerSentEventParserSpec extends BaseSpec {
   "A ServerSentEventParser" should {
 
     "parse ServerSentEvents correctly" in {
-      val input = """|data: message 1 line 1
-                     |data:message 1 line 2
-                     |
-                     |data: message 2
-                     |:This is a comment and must be ignored
-                     |event: Only the last event should be considered
-                     |event: message 2 event
-                     |
-                     |data:
-                     |
-                     |event: message 4 event
-                     |
-                     |data: incomplete message
-                     |""".stripMargin
+      val input = List(
+        "data: message 1 line 1",
+        "data:message 1 line 2",
+        "",
+        "data: message 2",
+        ":This is a comment and must be ignored",
+        "event: Only the last event should be considered",
+        "event: message 2 event",
+        "",
+        "data:",
+        "",
+        "event: message 4 event",
+        "",
+        "data: incomplete message",
+        "").mkString("\n")
```

Comment:
This fails the test.

When git clones the project, it magically replaces all LF in the source file with CR+LF. So the string 'input' suddenly has CR+LF's in it, causing the test to fail. Whereas on UNIX LFs stay as LFs.

The test below doesn't fail as there are no source newlines.